src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/RecommendTelemetryEvent.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using NuGet.Common;
+
+namespace NuGet.PackageManagement.Telemetry
+{
+    public class RecommendTelemetryEvent : TelemetryEvent
+    {
+        public RecommendTelemetryEvent(int numRecommended, double duration) : base("RecommendPackages")
```

Comment:
This feels like a very specific tel event. Can we attach it to some existing events if possible?

Ideally the recommender itself can measure its performance too.

Reply:
Moved this telemetry event to the recommender.